python/export_template.py

```diff
@@ -0,0 +1,33 @@
+from urllib import request
+import os
+
+
+root_dir = "discordfs-export"
+headers = {
+    "User-Agent": "discordfs/1.0"
+}
+
+files = [
+
+]
+
+if not os.path.exists(root_dir):
+    os.mkdir(root_dir)
+
+for f in files:
+    path = os.path.join(root_dir, f["channel_name"], f["filename"])
+    if os.path.exists(path):
+        f"Skipping {f['url']} (already downloaded)"
+        continue
+    print(f"Fetching {f['url']}")
+    req = request.Request(f["url"], {}, headers)
+    with request.urlopen(req) as url:
+        if url.status == 429:
+            print("Timeout")
```

Comment:
If it was a timeout, it would slow down while it's waiting for the response, no? We should still handle the retry_after response though to not anger Discord, I'll take care of it.

Reply:
I've implemented this, but right after I did, I started to wonder if this `retry_after` parameter is only included in API responses, and not when downloading files. I've left it in for now.

I'll test this out more later by copying and pasting one of the links in the script a few thousand times.

Reply:
I'll wrap it in a try-catch just in case for now.